hudi-spark-datasource/hudi-spark3/src/main/scala/org/apache/spark/sql/hudi/catalog/HoodieCatalog.scala

```diff
@@ -116,7 +116,10 @@ class HoodieCatalog extends DelegatingCatalogExtension
 
   override def tableExists(ident: Identifier): Boolean = super.tableExists(ident)
 
-  override def dropTable(ident: Identifier): Boolean = super.dropTable(ident)
+  override def dropTable(ident: Identifier): Boolean = {
+    DropHoodieTableCommand(ident.asTableIdentifier, ifExists = true, isView = false, purge = false).run(spark)
+    true
+  }
```

Comment:
@leesf there are other `super.dropTable()` usage in this class, should change them too?

Reply:
yes, updated